chat-widget/src/components/livechatwidget/common/defaultProps/dummyDefaultProps.ts

```diff
@@ -400,7 +400,8 @@ export const dummyDefaultProps: ILiveChatWidgetProps = {
                 alignItems: "center",
                 margin: "0px 14px 0px 14px",
                 textOverflow: "ellipsis !important",
-                width: "max-content",
```

Comment:
is this related to the bump?, otherwise , add it to the changes log and include test evidence

Reply:
That is related to this other PR https://github.com/microsoft/omnichannel-chat-widget/pull/206, yesterday Charlie told me that once I closed I had to bump the version and update the dummyDefualtProps